### DISCUSSION TITLE
alignment

### DISCUSSION THREAD

**Discussion by 14paavang** [valinet/ExplorerPatcher#497] [Ideas]
pls add a toggle to centre align the windows 10 taskbar, I had gotten used to it on windows 11 and even in windows 10, I kept the search big so all the apps were more or less in the centre

**Comment by BraINstinct0** [comment]
~~That is near impossible, since ExplorerPatcher is~~
- ~~Aimed primarily at reviving Windows 10 UI/UX components left dormant in Windows 11~~
- ~~Use Win10 components left in Win11, not recreate those (except for [Win10 Style Windows _Switcher](https://github.com/valinet/sws/))~~
- ~~Not a fully-fledged Windows personalization tool.~~

(pre)Edit: #43 seems to mention it (the third one), and seems like it isn't currently supported but will be in the long run.
But don't expect it to be too easy(for the reasons mentioned above) plus centering wasn't possible on Win10 so that would require some more hacks to achieve.

What I previously wrote above was based on my understanding of ExplorerPatcher(No, I'm not the dev so those are totally my personal thoughts.) but seems like I was wrong.

**Reply by 14paavang** [reply]
I am just asking for the full windows 10 taskbar to be available and not just the paired down version without the large search, basically, I would prefer the search icon in the taskbar to be bigger like stock windows 10 with a typing space so it will be closer to centre aligned

**Reply by BraINstinct0** [reply]
Ah, I misunderstood, so basically this is _not_ asking for direct centre-align functionality, but
1) displaying the 'search box'(white blank) next to search,
2) so it can 'push' icons to the center.
Is that correct?(re-wording to make it clearer for valinet to implement(if, valinet wants to, of course.))

**Reply by BraINstinct0** [reply]
FYI [adding an empty toolbar method](https://www.howtogeek.com/736658/how-to-center-your-windows-10-taskbar-icons-like-windows-11/) will work perfectly.(only caveat being start button left at the end.)

**Reply by valinet** [reply]
> I am just asking for the full windows 10 taskbar to be available and not just the paired down version without the large search, basically, I would prefer the search icon in the taskbar to be bigger like stock windows 10 with a typing space so it will be closer to centre aligned

The legacy Search box in the Windows 10 taskbar was broken by Microsoft in Windows 11. Not easily fixable, I don’t have access to the source code, everything is a million times more difficult and hacky than it could be. As a better replacement anyway, you can use [Everything Toolbar](https://github.com/stnkl/EverythingToolbar) which is basically the same thing only it is powered by Everything and uses that to search your computer. It will cover as much space as you want, as the original box, and also provides something MUCH more useful in return. Regular Windows Search can always be accessed by pressing the Start button or Windows key and starting typing anyway.

Thanks
